src/explore.sh

```diff
@@ -158,6 +161,22 @@ function parse_explore_options()
         ;;
       --verbose)
         options_values['VERBOSE']=1
+        shift
+        ;;
+      --show-context | -C)
+        if [[ -n "${options_values['CONTEXT']}" ]]; then
+          options_values['ERROR']='Invalid arguments: Multiple context values!'
+          return 22 # EINVAL
+        fi
+
+        # Check if the next argument exists and is an integer
+        if [[ ! "$2" =~ ^[0-9]+$ ]]; then
+          options_values['CONTEXT']=3
+        else
+          options_values['CONTEXT']="$2"
+          shift
```

Comment:
Don't you need to use `shift 2` since it can have a second item?

Reply:
There is a `shift` statement outside the else block. It adds up to have the same effect.